src/wms/src/test/java/org/geoserver/wms/LegendNonValidCapabilitiesTest.java

```diff
@@ -94,8 +94,7 @@ public void testCapabilities_1_3_0() throws Exception {
         assertXpathEvaluatesTo(LEGEND_FORMAT, legendUrlPath + "/wms:Format", dom);
         assertXpathEvaluatesTo(
                 BASE
-                        + "/ows?service=WMS&request=GetLegendGraphic&format=image%2Fpng&width=20"
-                        + "&height=20&layer=gs%3Awatertemp",
+                        + "/ows?service=WMS&request=GetLegendGraphic&format=image%2Fpng&width=20&height=20&layer=gs%3Awatertemp",
```

Comment:
Is there an actual change here? Looks like a reformat only.

Reply:
fixed